src/services/commit/useCommit.spec.tsx

```diff
@@ -135,6 +135,70 @@ const dataReturned = {
   },
 }
 
+const dataReturnedTeam = {
+  owner: {
+    repository: {
+      __typename: 'Repository',
+      commit: {
+        branchName: null,
+        totals: {
+          coverage: 38.30846,
+          diff: {
+            coverage: null,
+          },
+        },
+        commitid: 'f00162848a3cebc0728d915763c2fd9e92132408',
+        pullId: 10,
+        createdAt: '2020-08-25T16:35:32',
+        author: {
+          username: 'febg',
+        },
+        state: 'complete',
+        uploads: {
+          edges: [
+            {
+              node: {
+                id: 0,
+                state: 'PROCESSED',
+                provider: 'travis',
+                createdAt: '2020-08-25T16:36:19.55947400:00',
+                updatedAt: '2020-08-25T16:36:19.67986800:00',
+                downloadUrl:
+                  '/api/gh/febg/repo-test/download/build?path=v4/raw/2020-08-25/F84D6D9A7F883055E40E3B380280BC44/f00162848a3cebc0728d915763c2fd9e92132408/30582d33-de37-4272-ad50-c4dc805802fb.txt',
+                ciUrl: 'https://travis-ci.com/febg/repo-test/jobs/721065746',
+                uploadType: 'UPLOADED',
```

Comment:
Another note I just realized, if the session comes with uploadType `carryforward`, will we show that session altogether as well? I forget specifically what we want to do in the interim

Reply:
Yeah it will be shown(without any mention of flags in the uploads pane). However there are no customers in the team plan with that configuration. 

When we implement more foundational changes, I expect fetching those to be skipped at the API level.